mission/functions/systems/consumables/fn_consume.sqf

```diff
@@ -98,6 +99,7 @@ if (isClass _item_config) then
 		};
 	};
 
+	// @dijksterhuis remoteExec here
 	// change stats related to ammount of item consumed
 	["eatdrink", [_water,_food,_item]] call para_c_fnc_call_on_server;
```

Comment:
I need to double check exactly what these scripts are from the `rehandler.hpp` config file

Reply:
there's also going to be more of them elsewhere

Reply:
list from `rehandler.hpp` so i don;t have to go and look it up again

```sqf
vn_mf_fnc_arsenal_trash_cleanup
vn_mf_fnc_veh_asset_handle_change_vehicle_request
vn_mf_fnc_changeteam
vn_mf_fnc_eatdrinkvn_mf_fnc_packageforslingloading
vn_mf_fnc_settrait
vn_mf_fnc_supplyrequest
vn_mf_fnc_supporttaskcreate
vn_mf_fnc_teleport
```